nexus/reconfigurator/planning/src/planner.rs

```diff
@@ -214,17 +214,18 @@ impl<'a> Planner<'a> {
         // The sled is not expunged. We have to see if the inventory
         // reflects the parent blueprint disk generation. If it does
         // then we mark any expunged disks decommissioned.
+        //
+        // TODO-correctness We inspect `last_reconciliation` here to confirm
```

Comment:
On balance, this seems like the right choice to me. We should know the sled agent has acted before decommissioning.